package.json

```diff
@@ -7,8 +7,9 @@
   "scripts": {
     "check-coverage": "istanbul check-coverage",
     "coverage": "istanbul cover ./node_modules/.bin/_mocha && istanbul report cobertura",
+    "docs": "for v in 1.7 1.8 1.9; do scripts/docs.js --spec lib/specs/swagger-$v.json.gz --output docs/$v.md; done",
```

Comment:
This won't work cross-platform.

Reply:
For my own learning - which part will break? I'm assuming on Windows?

Reply:
1. bash-style for-loop syntax won't work on Windows (syntax error)
1. executing scripts/docs.js directly won't work on Windows (not an executable)